waspc/data/Generator/templates/server/src/jobs/_job.js

```diff
@@ -2,4 +2,9 @@
 import { createJob } from './{= executorJobRelFP =}'
 {=& jobPerformFnImportStatement =}
 
-export const {= jobName =} = await createJob({ jobName: "{= jobName =}", jobFn: {= jobPerformFnName =}, defaultJobOptions: {=& jobPerformOptions =} })
+export const {= jobName =} = await createJob({
+  jobName: "{= jobName =}",
+  jobFn: {= jobPerformFnName =},
+  defaultJobOptions: {=& jobPerformOptions =},
+  jobSchedule: {=& jobSchedule =}
+})
```

Comment:
I am interested, but what is your reasoning behind that?

Reply:
Just to reduce the number of files. The project is easier to navigate, and the job definitions are pretty small.

Reply:
Although, ideally, IMO all projects should be split vertically (by grouping related features/functionalities) instead of horizontally (by grouping similar code like services, controllers, components).

However, it would be really difficult to do this with Wasp, and since we the stuff is grouped horizontally already, we might as well reduce the number of files.

Reply:
Sure, sounds reasonable! We can also leave this for the future and then do it both for jobs, queries, and maybe smth else.

Reply:
OK, leaving this one for the future!